components/parameters/StorageBucket.yaml

```diff
@@ -0,0 +1,7 @@
+name: bucket
+in: path
+required: true
+description: name and prefix of the bucket integration
```

Comment:
```suggestion
description: name of the bucket integration
```

Reply:
There may be a prefix, there may not be. This is what I'm trying to convey here.
Is
"name of the bucket integration (and prefix if specified)"
better?